packages/gatsby-plugin-offline/README.md

```diff
@@ -29,34 +29,42 @@ and AppCache setup by changing these options so tread carefully.
 
 ```javascript
 const options = {
-  staticFileGlobs: [
-    `${rootDir}/**/*.{woff2}`,
-    `${rootDir}/commons-*js`,
-    `${rootDir}/app-*js`,
+  staticFileGlobs: files.concat([
     `${rootDir}/index.html`,
     `${rootDir}/manifest.json`,
     `${rootDir}/manifest.webmanifest`,
     `${rootDir}/offline-plugin-app-shell-fallback/index.html`,
-  ],
+    // Webpack chunks aren't always detected by `getResourcesFromHTML`
+    // since they're sometimes loaded from other chunks.
+    `${rootDir}/+([1234567890])-*.js`,
+    ...criticalFilePaths,
+  ]),
   stripPrefix: rootDir,
+  // If `pathPrefix` is configured by user, we should replace
+  // the `public` prefix with `pathPrefix`.
+  // See more at:
+  // https://github.com/GoogleChrome/sw-precache#replaceprefix-string
+  replacePrefix: args.pathPrefix || ``,
   navigateFallback: `/offline-plugin-app-shell-fallback/index.html`,
-  // Only match URLs without extensions.
+  // Only match URLs without extensions or the query `no-cache=1`.
```

Comment:
where's this "no-cache=1" coming from?

Reply:
oh, I see I think. You're using it as a way to to ensure the sw does catch the reload?

Reply:
It's for when Gatsby detects a page not found after loading from the offline shell - usually it's correct, but in some cases the website could've been updated, or it's a static file which isn't handled by Gatsby (e.g. Netlify CMS's /admin/ page). We need to check if these files actually exist, which means they need to be loaded from the server directly, not from the cache - that's the purpose of this query, to prevent the cache handling them.